prog/setup_spdk.rb

```diff
@@ -10,7 +10,12 @@ def start
   end
 
   def start_service
+    sshable.cmd("sudo systemctl enable home-spdk-hugepages.mount")
+    sshable.cmd("sudo systemctl enable spdk")
+
+    sshable.cmd("sudo systemctl start home-spdk-hugepages.mount")
```

Comment:
Do you need to do this yourself? Since the unit file could track the dependency.